1_space_odyssey/js/oda-9000/app.js

```diff
@@ -105,19 +105,75 @@ var isInGoodMood = true;
 console.log(".-.-. Booting .-.-.");
 console.log(name, isAlive, hasJetPack, eyes, eyeColor, hairColor, legs, arms, hands, ears, canWalk, canFly, hasWeapon);
 
-if (hasJetPack) {
-  console.log("JetPack is enabled!!! off I go outside to fix the satellite");
-} else {
-  console.log("JetPack NOT enabled, please enable jetPack before doing a space walk");
-}
 /*
  2. BROKEN SATELLITE::::::
     You need to send the droid out on a space walk to fix the satellite, with out it you can't communicate with earth.
     a) write an if condition checking if the jet pack is enabled, if true the droid can go outside if false the droid
        cannot go outside, console log the correct log based on that condition.
 */
+console.log(
+  "\u{1F6F0} BROKEN SATELLITE::::::"
+);
+
+if (hasJetPack) {
+  console.log("JetPack is enabled!!! off I go outside to fix the satellite");
+} else {
+  console.log("JetPack NOT enabled, please enable jetPack before doing a space walk");
+}
 
 /*
- uncomment these console logs and write your if else statement and use them to console log out the correct statement
 
- */
+/*
+ 3. METEOR SHOWER::::::
+    On your way to Jupiter you encounter a massive meteor shower that could damage the ship, you need to redirect the ship to a new safe location.
+    a) Using a for loop iterate through the responses array and choose the appropriate response to get out of danger.
+      create an if condition in your loop to print only 1 response using a comparison operator.
+*/
+console.log(
+  "\u{1F327} METEOR SHOWER::::::"
+);
+
+var responses = [
+  "Call basecamp and ask for assistance, but they are far away",
+  "Do nothing, just push through",
+  "initiate core thrusters, punch 180degrees into dash-nav and warp away from danger",
+  "ask humanoid-9000 to fly out there and shield the ship from the debris, this action could harm the humanoid"
+];
+
+// write your code here, use the console log beneath
+// console.log("\u{1F6F8} Swoosh -> -> Ship is steered to safety!");
+
+console.log(responses[2])
+for (var i = 0; i < responses.length; i++) {
+  if (i === 2) {
```

Comment:
short and sweet. 

another option would be to use index, in some case this option might seem more readable at first glance.
```js
i === responses[2]
```